public/js/index.js

```diff
@@ -0,0 +1,24 @@
+//Youtube slideshow for index page
+function onGoogleLoad() {
+  gapi.client.setApiKey('AIzaSyCzI5Gvpq1hPlfjvB81EVc0vLv_11eW7NI');
```

Comment:
It's not ideal, but if google's javascript library requires an apikey, we can only presume that it's intended for the key to be public